src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -42,7 +42,7 @@ export interface ModelInputsPopulationBrackets {
 }
 interface ModelInputsPersistent extends ModelInputsPopulationBrackets {
   facilityDormitoryPct?: number;
-  facilityOccupancyPct?: number;
+  facilityCapacity?: number | undefined;
```

Comment:
I think technically you don't need the extra `undefined`, the `?` covers that case. Similarly, on line 64, if the field can be undefined you don't need to add it there at all; that object is really just for making keys required that were optional in the interfaces it inherits from (convoluted, I know ... an artifact of my learning curve with TypeScript).

Reply:
Thanks for pointing that out. I'm also still trying to learn TypeScript...sometimes it feels like unraveling a big ball of string. I ended up making a couple of other changes here as I really thought about what the different interfaces were being used for and how the undefined default is handled.